jwala-services/src/main/java/com/cerner/jwala/service/webserver/impl/WebServerServiceImpl.java

```diff
@@ -50,17 +53,21 @@
 
     private final BinaryDistributionLockManager binaryDistributionLockManager;
 
+    private final JvmPersistenceService jvmPersistenceService;
+
     public WebServerServiceImpl(final WebServerPersistenceService webServerPersistenceService,
                                 final ResourceService resourceService,
                                 @Qualifier("webServerInMemoryStateManagerService")
                                 final InMemoryStateManagerService<Identifier<WebServer>, WebServerReachableState> inMemoryStateManagerService,
                                 final String templatePath,
-                                final BinaryDistributionLockManager binaryDistributionLockManager) {
+                                final BinaryDistributionLockManager binaryDistributionLockManager,
```

Comment:
I would prefer all new construction injection parameters be auto wired. But like Jedd said earlier, this might require a lot of re-work in the unit test.